### PR TITLE
Add HiC

### PR DESCRIPTION
**Pull request by ningbioinfostruggling**
**https://github.com/ningbioinfostruggling/awesome-HiC**

High-resolution chromosome conformation capture sequencing (HiC-seq) has been one of the most important methods to investigate the 3D genome since its first publication in 2009. Since then, numerous of softwares and packages have been developed to facilitate the analysis of HiC-seq data. This collection of awesome HiC softwares can help researchers who are interesting in analysing HiC-seq data or developing similar softwares.

And I've reviewed a couple of PRs and here is my awesome-lint check:
```
 $ awesome-lint https://github.com/ningbioinfostruggling/awesome-HiC
✖ Linting

  README.md:13:1
  ✖  13:1  ToC item "Pipelines" link "#Pipelines" not found  awesome/toc

  1 error
```
I don't understand why I have this error because the link of that TOC is totally working.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by sindresorhus** [comment]
Would be better to make the title of the list `Awesome High-Resolution Chromosome`.

File: readme.md [hunk 1]
```diff
@@ -605,7 +605,7 @@
 - [Speakers](https://github.com/karlhorky/awesome-speakers#readme) - Conference and meetup speakers in the programming and design community.
 - [Software Patreons](https://github.com/uraimo/awesome-software-patreons#readme) - Fund individual programmers or the development of open source projects.
 - [Parasite](https://github.com/ecohealthalliance/awesome-parasite#readme) - Parasites and host-pathogen interactions.
-
+- [HiC](https://github.com/ningbioinfostruggling/awesome-HiC#readme) - HiC-seq data related softwares/packages/intro-papers.
```
Comment (sindresorhus) [inline readme.md]: The entry in the Awesome list should:
* Include a short description about the project/theme of the list. **It should not describe the list itself.**

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/ningbioinfostruggling/awesome-HiC
✖ Linting

  README.md:13:1
  ✖  13:1  ToC item "Pipelines" link "#Pipelines" not found  remark-lint:awesome/toc

  1 error
```

https://github.com/sindresorhus/awesome-lint

**Comment by ningbioinfostruggling** [comment]
> Would be better to make the title of the list `Awesome High-Resolution Chromosome`.

Hi, thanks for the suggestion but the full name of HiC is High-resolution Chromosome Conformation Capture, but it would be too long to use the full name and I believe using Awesome HiC can be much easier to be found by people with interest.

**Comment by ningbioinfostruggling** [comment]
> ```
> ❯ awesome-lint https://github.com/ningbioinfostruggling/awesome-HiC
> ✖ Linting
> 
>   README.md:13:1
>   ✖  13:1  ToC item "Pipelines" link "#Pipelines" not found  remark-lint:awesome/toc
> 
>   1 error
> ```
> https://github.com/sindresorhus/awesome-lint

Yeah I try that and have included that in my PR, I don't understand why I have this error because the link of that TOC is totally working.

**Comment by sindresorhus** [comment]
I would recommend taking the text in your list through something like Grammarly. There are a lot of typos.

`three_dimensional` => `three-dimensional`

`numerous of softwares` => `numerous of software`

`where physical contacts` => `where physical contact`

`Awesome collections of HiC-seq analysing softwares` => `Curation of awesome HiC-seq analysis software`

---

> The eukaryotic genome have hierarchical 

I would linkify `eukaryotic genome` to https://www.nature.com/scitable/topicpage/eukaryotic-genome-complexity-437 or another site that explains it well.

File: readme.md [hunk 2]
```diff
@@ -616,7 +616,7 @@
 - [Amazon Sellers](https://github.com/ScaleLeap/awesome-amazon-seller#readme)
 - [Agriculture](https://github.com/beaorn/awesome-agriculture#readme) - Open source technology for farming and gardening.
 - [Plotters](https://github.com/beardicus/awesome-plotters#readme) - Computer-controlled drawing machines and other visual art robots.
-
+- [HiC](https://github.com/ningbioinfostruggling/awesome-HiC#readme) - High-resolution Chromosome Conformation Capture Sequencing.
```
Comment (sindresorhus) [inline readme.md]: This should describe what `HiC` is in a succinct manner.

Something like:

> One of the most important methods to investigate the eukaryotic genome.

**Comment by sindresorhus** [comment]
> The repo should have awesome-list & awesome as GitHub topics. I encourage you to add more relevant topics.

Please ensure you've read all the guidelines carefully.

**Pull request closed without merging**